company/remote/index.md

```diff
@@ -53,6 +53,7 @@ After that it is just a matter of not checking your work email (or Slack) at tim
 
 - [Our journey to all-remote](https://about.sourcegraph.com/blog/our-journey-to-all-remote)
 - [Tips for working remotely](tips.md)
+- [Sourcegraph Social Calendar](calendar.md)
```

Comment:
```suggestion
- [Social calendar](calendar.md)
```

https://about.sourcegraph.com/handbook/communication/style_guide#sentence-case

Reply:
I have changed the case in all of them now, but I chose to use that casing as it is the calendar name and not just a calendar that is called Social.